mkdocs/docs/configuration.md

```diff
@@ -30,15 +30,23 @@ Iceberg tables support table properties to configure table behavior.
 
 ### Write options
 
-| Key                                    | Options                           | Default | Description                                                                                 |
-| -------------------------------------- | --------------------------------- | ------- | ------------------------------------------------------------------------------------------- |
-| `write.parquet.compression-codec`      | `{uncompressed,zstd,gzip,snappy}` | zstd    | Sets the Parquet compression coddec.                                                        |
-| `write.parquet.compression-level`      | Integer                           | null    | Parquet compression level for the codec. If not set, it is up to PyIceberg                  |
-| `write.parquet.row-group-limit`        | Number of rows                    | 1048576 | The upper bound of the number of entries within a single row group                          |
-| `write.parquet.page-size-bytes`        | Size in bytes                     | 1MB     | Set a target threshold for the approximate encoded size of data pages within a column chunk |
-| `write.parquet.page-row-limit`         | Number of rows                    | 20000   | Set a target threshold for the approximate encoded size of data pages within a column chunk |
-| `write.parquet.dict-size-bytes`        | Size in bytes                     | 2MB     | Set the dictionary page size limit per row group                                            |
-| `write.metadata.previous-versions-max` | Integer                           | 100     | The max number of previous version metadata files to keep before deleting after commit.     |
+| Key                                    | Options                                | Default         | Description                                                                                 |
+| -------------------------------------- | -------------------------------------- | --------------- | ------------------------------------------------------------------------------------------- |
+| `write.parquet.compression-codec`      | `{uncompressed,zstd,gzip,snappy}`     | zstd            | Sets the Parquet compression codec.                                                         |
+| `write.parquet.compression-level`      | Integer                                | null            | Parquet compression level for the codec. If not set, it is up to PyIceberg.                 |
+| `write.parquet.row-group-limit`        | Number of rows                         | 1,048,576       | The upper bound of the number of entries within a single row group.                         |
+| `write.parquet.row-group-size-bytes`   | Size in bytes                          | 128 MB          | The maximum size (in bytes) of each Parquet row group.                                      |
```

Comment:
This one is not supported: https://github.com/apache/iceberg-python/blob/583a7e97db28afa4259cdf504611845222338893/pyiceberg/io/pyarrow.py#L2550-L2556

We can also make that explicit in the docs.